prow/cluster/jobs/istio/test-infra/istio.test-infra.master.yaml

```diff
@@ -36,6 +36,7 @@ presubmits:
     context: "prow: build-test"
     always_run: true
     decorate: true
+    branches: *branch_spec
```

Comment:
I'd vote for dropping this part and running on all branches

Reply:
Done.